lib/rules/space-in-parens.js

```diff
@@ -184,9 +188,9 @@ module.exports = function(context) {
 
     return {
         "Program": function checkParenSpaces(node) {
-            var tokens = node.tokens;
-            var prevToken, nextToken;
+            var tokens, prevToken, nextToken;
             exceptions = getExceptions();
+            tokens = sourceCode.tokensAndComments;
```

Comment:
In any event, I think this is a good approach. If it ends up being suboptimal, I'm to blame. :)